EU4toV2/Data_Files/blankMod/output/history/pops/1836.1.1/Belize.txt

```diff
@@ -3,7 +3,7 @@
 #Belize (7200/1800 POPS)
 2190 = {
 	artisans = {
-		culture = british
+		culture = texan
```

Comment:
These are only relevant is the province converts unowned, so a non-colonial culture should be chosen instead. Or if the province is always owned due to EU4 startin conditions, it should remain the default for easier comparisons to the normal Vic2 file.

Reply:
This didn't happened to be a problem when Gypsies and Macedonians were introduced in the files. Although, I can see why using "Texan" may not be a good choice. British culture, in the EU4 conversions, happens to lack anything that could link it with playgame experience, as now split between English and Scottish.

Reply:
Yeah, I should have been more attentive then. Tryjng to keep things cleaner now, though.

Reply:
So, what do I have to do?

Reply:
Please go through the changed files and:
1. If the province is always owned, revert your changes. The converter is going to change the culture anyways.
2. If you feel like it, for provinces that might be unowned on conversion, change the culture to something appropriate for that part of the world instead of a colonizer. Not strictly necessary, as your current changes aren't making things worse, but by doing so you'd make it better.

Let me know when you've done so and I'll just trust you did good work. I got a bit micro-managing with this review, and I didn't mean to.

Reply:
Done.